modules/rest-api/pages/rest-name-node.adoc

```diff
@@ -59,7 +59,7 @@ http://10.144.220.101:8091/node/controller/rename \
 
 The sequence of tasks divided into _initialization_ and _provisioning_ is explained in xref:rest-api:rest-cluster-init-and-provisioning.adoc[Cluster Initialization and Provisioning].
 
-For each of the other specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
+For each of the other specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
```

Comment:
For each of the other specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].